### DISCUSSION TITLE
Bitbucket Cloud SSH Host Keys

### DISCUSSION THREAD

**Discussion by andriusAtXdesign** [codemagic-ci-cd/codemagic-docs#1776] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Other - please mention in your question

### Steps to reproduce

Based on new blog https://bitbucket.org/blog/ssh-host-key-changes posted by Bitbucket do users need to take any actions or Codemagic will/has handled it internally?

### Expected results

Adhere to changes mentioned in https://bitbucket.org/blog/ssh-host-key-changes

### Actual results

N/A

**Comment by dtrdic** [comment, marked as answer]
hello @andriusAtXdesign, no action is required for Bitbucket Pipelines builds or user-generated SSH keys, as the builds start from a clean VM without any Bitbucket keys. However, considering the security aspect, it may be a good idea to update your SSH keys as explained in the official Bitbucket announcement

**Reply by andriusAtXdesign** [reply]
Thank you!
